Formula/streamsets.rb

```diff
@@ -0,0 +1,50 @@
+class Streamsets < Formula
+  desc "ETL tool with a graphical IDE for building dataflow pipelines"
+  homepage "https://streamsets.com/"
+  url "https://archives.streamsets.com/datacollector/1.5.1.0/tarball/streamsets-datacollector-all-1.5.1.0.tgz"
+  sha256 "d848ffaec4cbadce0e7aa031fb30d1e7e13c84be9115ecd6c69643ff02400fc8"
+
+  bottle :unneeded
+
+  depends_on :java => "1.7+"
+  depends_on "md5sha1sum"
```

Comment:
Any chance we can talk nicely to upstream here & see if they're willing to accommodate making their code more cross-platform friendly? OS X/macOS does have a fine checksum utility (`shasum`), it just isn't named `sha1sum`.